packages/config-plugins/src/plugins/mod-compiler.ts

```diff
@@ -164,9 +165,11 @@ export async function evalModsAsync(
           if (assertMissingModProviders !== false) {
             throw new PluginError(errorMessage, 'MISSING_PROVIDER');
           } else {
-            if (config._internal?.isDebug) {
-              console.warn(errorMessage);
-            }
+            Warnings.addWarningForPlatform(
+              platformName as ModPlatform,
+              `${platformName}.${modName}`,
+              `Skipping: Initial base modifier for "${platformName}.${modName}" is not a provider and therefore will not provide modResults to child mods`
```

Comment:
```suggestion
              `Skipping: Initial base modifier for "${platformName}.${modName}" is not a provider and therefore will not provide modResults to child mods. This may be due to an outdated version of Expo CLI.`
```

Reply:
Could be problematic if you aren't using expo-cli